hack/lib/golang.sh

```diff
@@ -480,6 +480,7 @@ EOF
   local go_version
   IFS=" " read -ra go_version <<< "$(GOFLAGS='' go version)"
   local minimum_go_version
+  # TODO(liggitt): Need to switch this to 1.19 once we update images to newer go version
```

Comment:
needs to switch?

Reply:
yeah, I'll update this once we switch presubmit images to run with go1.19, it would have turned all CI jobs red before that

Reply:
Actually, I think we should avoid bumping this until we're happy with post-submits and maybe until we cut the next 1.24 patch release. As soon as we do this, anything currently happily building off of release-1.24 with go1.18 will start failing, and I don't want to ripple out build/config changes past the immediate k8s.io/test-infra and k8s.io/release updates we're making quite yet

Reply:
opened https://github.com/kubernetes/kubernetes/pull/114616 with this follow-up